index.html

```diff
@@ -87,7 +87,7 @@
           -->
         <video id=example-video width=960 height=400 class="video-js vjs-default-skin" controls>
           <source
-             src="//d1sh0fycs31s5a.cloudfront.net/eleph/index.m3u8"
+             src="https://s3.amazonaws.com/_bc_dml/example-content/bipbop-advanced/bipbop_16x9_variant.m3u8"
```

Comment:
you should use the cloudfront url for this.
`//d2zihajmogu5jn.cloudfront.net/bipbop-advanced/bipbop_16x9_variant.m3u8`